examples/tf/her_ddpg_fetchreach.py

```diff
@@ -71,7 +71,7 @@ def run_task(*_):
 
         runner.setup(algo=ddpg, env=env)
 
-        runner.train(n_epochs=50, n_epoch_cycles=20)
+        runner.train(n_epochs=50, n_epoch_cycles=20, batch_size=100)
 
 
 run_experiment(run_task, snapshot_mode='last', seed=1)
```

Comment:
why would run_experiment still require a `snapshot_mode` param if LocalRunner requires a `snapshot_config`?

Reply:
Because `snapshot_config` is constructed inside `experiment_wrapper`. And we pass the parameters (str, mode, gap...) to it using `run_experiment`.